alluxiofs/client/core.py

```diff
@@ -119,14 +117,8 @@ class AlluxioClient:
 
     def __init__(
         self,
-        etcd_hosts=None,
-        worker_hosts=None,
-        options=None,
-        concurrency=64,
-        etcd_port=2379,
-        worker_http_port=ALLUXIO_WORKER_HTTP_SERVER_PORT_DEFAULT_VALUE,
-        etcd_refresh_workers_interval=120,
         test_options=None,
```

Comment:
emmm, i want to make it invisible, but currently it's too visible, can we put it in **kwargs, as well